src/test/resources/io/fabric8/elasticsearch/plugin/project_strategy_roles_shared_non_ops_kibana_mode.yml

```diff
@@ -11,9 +11,9 @@ gen_ocp_kibana_shared_non_ops:
 gen_project_foo_bar:
   expires: '10'
   indices:
-    ?project?foo?bar?*:
+    ?project?foo?bar?123abc?*:
```

Comment:
in this case - `project` is the project prefix, `foo` is the namespace name?  what is `bar`?  is `123abc` the uuid?